components/generic/CompareImageSlider.vue

```diff
@@ -155,6 +156,18 @@
         this.dragging = false;
       },
 
+      showHideAttribution() {
+        if (this.dragging) {
+          if (this.sliderPosition < 0.2) {
+            this.hideAttribution = 'left';
+          } else if (this.sliderPosition > 0.8) {
```

Comment:
I'm wondering if the 0.2 and 0.8 should be dynamic, as these will change when you reduce the size of the browser window ie Mobile.

Reply:
I kept the values static for now. And moved the attributions to below the images for small screens.